src/main/java/cms/gov/madie/terminology/config/SecurityConfig.java

```diff
@@ -0,0 +1,35 @@
+package cms.gov.madie.terminology.config;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.security.config.annotation.web.builders.HttpSecurity;
+import org.springframework.security.config.annotation.web.configuration.WebSecurityConfigurerAdapter;
+import org.springframework.security.config.http.SessionCreationPolicy;
+
+@Configuration
+public class SecurityConfig extends WebSecurityConfigurerAdapter {
```

Comment:
@gregory-akins , @jkotanchik-SB does this service need spring security? no data storage, read-only operations, and already has VSAC authentication.

Reply:
@adongare @gregory-akins @jkotanchik-SB I thought madie would call this service and this service would have VSAC authentication to call UMLS? Maybe I misunderstand...

Reply:
No, you got it right. 
My question was- do we need our own authentication(spring security)? I was asking this because we don't have Database for this service. We are just reading valuset data from VSAC service which already has authentication in place. Was trying to understand the benefits of adding spring security.

Reply:
It's likely we will be using a database for storing the UMLS key. SecOps has a strat that I'm proofing now that will permit this.

The same security controls applied to `measure-service` and `cql-library-service` should also be applied here.

Reply:
@jkotanchik-SB @adongare @gregory-akins Ok thanks all for your help!